src/main/kotlin/ch/hevs/cloudio/cloud/services/DelayedService.kt

```diff
@@ -0,0 +1,118 @@
+            val data = message.body
+            val messageFormat = JsonSerializationFormat.detect(data)
+            if (messageFormat) {
+
+                val delayed = DelayedContainer()
+
+                JsonSerializationFormat.deserializeDelayed(delayed, data)
+
+                delayed.messages.forEach { delayedMessage ->
+
+                    val splitMessage = delayedMessage.topic.split("/")
+                    val prefix = splitMessage[0]
+                    when(prefix){
+                        "@update" -> {
+                            val innerData = delayedMessage.data
+                            if(innerData is LinkedHashMap<*, *>) {
+                                val attribute = mapper.convertValue(innerData, Attribute::class.java)
+                                handleUpdate(delayedMessage.topic, attribute)
+                            }
+                        }
+                        "@transaction" -> {
+                            val innerData = delayedMessage.data
+                            if(innerData is LinkedHashMap<*, *>) {
+                                val transaction = mapper.convertValue(innerData, Transaction::class.java)
+                                handleTransaction(delayedMessage.topic, transaction)
+                            }
+                        }
+                        "@nodeAdded" -> {
+                            val innerData = delayedMessage.data
+                            if(innerData is LinkedHashMap<*, *>) {
+                                val node = mapper.convertValue(innerData, Node::class.java)
+                                handleNodeAdded(delayedMessage.topic, node)
+                            }
+                        }
+                        "@nodeRemoved" -> {
+                            handleNodeRemoved(delayedMessage.topic)
+                        }
+                        "@logs" -> {
+                            val innerData = delayedMessage.data
+                            if(innerData is LinkedHashMap<*, *>) {
+                                val log = mapper.convertValue(innerData, CloudioLogMessage::class.java)
+                                handleLogs(delayedMessage.topic, log)
+                            }
+                        }
+                        else -> {
+                            log.error("Unrecognized message with sub-topic ${delayedMessage.topic} " +
+                                    "inside @delayed message from $endpointId")
+                        }
+                    }
+                }
+            } else {
+                log.error("Unrecognized message format in @delayed message from $endpointId")
+            }
+        } catch (exception: Exception) {
+            log.error("Exception during @delayed message handling:", exception)
+        }
+    }
+
+    fun handleUpdate(topic: String, attribute: Attribute){
+        //TODO
```

Comment:
Is this intentionally not implemented?

Maybe add a log warn message to simplify testing.

Reply:
from my PR message:
> Added service handling them, due to database migration, I didn't complete the logic of this service, only the parsing of the message

Reply:
Ok, makes sense then.